genai_bench/cli/cli.py

```diff
@@ -77,6 +81,28 @@ def benchmark(
     batch_size,
     traffic_scenario,
     additional_request_params,
+    # Model auth options
```

Comment:
nit: Maybe it is time to introduce `kwargs` for this function. It is growing so fast. We can create an issue for this